src/handles/hstmt.rs

```diff
@@ -75,6 +75,13 @@ impl<'env, 'param> HStmt<'env> {
         ret.map(|()| out)
     }
 
+    pub fn affected_row_count(&self) -> Return<SQLLEN> {
+        let mut out: SQLLEN = 0;
+        let ret = unsafe { SQLRowCount(self.handle, &mut out) };
+        let ret: Return<()> = ret.into();
```

Comment:
Purely a style issue, but looks ugly. But I guess `ret.into().map(|()| out)` is not possible due to insufficient type inference?

Reply:
@adamreichold Unfortunately, yes. It looks good, but causes

> error[E0282]: type annotations needed
>   --> src/handles/hstmt.rs:81:9
>    |         ret.into().map(|()| out)
>    |         ^^^^^^^^^^ cannot infer type for `T`

Reply:
@adamreichold Unfortunately, yes. It looks good, but causes

> error[E0282]: type annotations needed
>   --> src/handles/hstmt.rs:81:9
>    |         ret.into().map(|()| out)
>    |         ^^^^^^^^^^ cannot infer type for `T`